cityofaustin/atd-vz-data
# Remove "Search by" from the grid table search menu items

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/18623

## Testing
**URL to test:** 

**Steps to test:**
1. See the new search labels on the crashes, locations and location details page tables.


---
#### Ship list
